Quadriga/src/main/java/edu/asu/spring/quadriga/dao/impl/workspace/ListExternalWsDAO.java

```diff
@@ -0,0 +1,62 @@
+package edu.asu.spring.quadriga.dao.impl.workspace;
+
+import java.util.List;
+
+import org.apache.commons.collections.CollectionUtils;
+import org.hibernate.HibernateException;
+import org.hibernate.Query;
+import org.hibernate.SessionFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+import edu.asu.spring.quadriga.dao.impl.BaseDAO;
+import edu.asu.spring.quadriga.dao.workspace.IListExternalWsDAO;
+import edu.asu.spring.quadriga.dto.ExternalWorkspaceDTO;
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+
+@Repository
+public class ListExternalWsDAO extends BaseDAO<ExternalWorkspaceDTO> implements IListExternalWsDAO {
+
+    @Autowired
+    private SessionFactory sessionFactory;
+
+    /**
+     * {@inheritDoc}
+     */
+    @SuppressWarnings("rawtypes")
+    @Override
+    public boolean isExternalWorkspaceExists(String workspaceId) throws QuadrigaStorageException {
```

Comment:
instead of this just create a method in the manager class and get workspace by id. If it is null, it doesn't exist.